tests/integration/test_etcd.py

```diff
@@ -40,4 +40,6 @@ async def test_etcd_datastore(kubernetes_cluster: model.Model):
     status = json.loads(result.results["stdout"])
     assert status["ready"], "Cluster isn't ready"
     assert status["datastore"]["type"] == "external", "Not bootstrapped against etcd"
-    assert status["datastore"]["external-url"] == f"https://{etcd.public_address}:{etcd_port}"
+    assert status["datastore"]["servers"] == [
```

Comment:
how can we test that the datastore config update works as expected? @mateoflorido is there an easy way to add an etcd member while the test is running?

Reply:
You can add a new test in the `test_etcd.py` module; something like `def test_update_etcd_cluster()`. For example, you could do something like [this](https://github.com/canonical/k8s-operator/blob/main/tests/integration/test_k8s.py#L98). Get the application from the model, and add an unit. Then you should verify that your etcd server's string has been updated.

Reply:
Thanks, added an integration test with this.